README.md

```diff
@@ -2,7 +2,7 @@
 
 [![doc](https://img.shields.io/badge/Doc-purple)](Documentation)  [![SPM](https://img.shields.io/badge/SPM-compatible-4BC51D.svg?style=flat)](https://github.com/apple/swift-package-manager/)
 
-The ArcGIS Maps SDK for Swift Toolkit contains components that will simplify your iOS app development. It is built off of the new ArcGIS Maps SDK for Swift.
+The ArcGIS Maps SDK for Swift Toolkit contains components that will simplify your Swift app development. It is built off of the new ArcGIS Maps SDK for Swift.
```

Comment:
I don't think this needs to be changed… 🤔

Reply:
The doc team made the change in the online doc, so I'm just duplicating that change.

Reply:
Here it says "that will help simplify your iOS app development"

https://developers.arcgis.com/swift/#open-source-toolkit (next site as well)

Reply:
I'm referring to the section description on the landing page, not the toolkit specific page. But I don't think it is a big deal in case you want to change them all to "Swift app development".